devel/pulp/devel/test_runner.py

```diff
@@ -12,6 +12,27 @@
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pulp.server.webservices.settings'
 
 
+# From: http://stackoverflow.com/questions/377017/test-if-executable-exists-in-python/377028#377028
```

Comment:
After getting some advice from legal, I need to ask for a new, original implementation. Since Pulp is distributed as GPLv2 and this is licensed as CC-A-SA to accept this implementation would cause Pulp to distribute under more than just GPLv2. Since its only one function, it's better to rewrite it as a GPLv2 implementation than to have Pulp distribute using a plurality of licenses.